benchmarks/results/158-pruning/sdk-pruning-1-0-results.txt

```diff
@@ -0,0 +1,164 @@
+Starting SimulateFromSeed with randomness created with seed 53
+Randomized simulation params: 

```

Comment:
Running the sdk simul is probably important and the right thing to do. Not sure these files should go into the iavl repo though. It would be good to summarize this somewhere. And when the sdk catches up with the iavl version, the sdk team can add the benchmarks (or a condensed summary) in the sdk repo (or sth short in the changelog and a description how to reproduce).

Reply:
I believe @AdityaSripal and I briefly decided that if there is a PR that is claiming improvements then benchmarks should be added, then upon doing a release a final benchmark will be created prior to release and saved in the repo while the others are deleted. I will document this